--- a/vendor/github.com/sevlyar/go-daemon/daemon_unix.go
+++ b/vendor/github.com/sevlyar/go-daemon/daemon_unix.go
@@ -134,9 +134,11 @@
 			return err
 		}
 		if d.pidFile, err = OpenLockFile(d.PidFileName, d.PidFilePerm); err != nil {
+			err = fmt.Errorf("OpenLockFile: %s", err.Error())
 			return
 		}
 		if err = d.pidFile.Lock(); err != nil {
+			err = fmt.Errorf("Lock: %s %s", d.PidFileName, err.Error())
 			return
 		}
 		if len(d.Chroot) > 0 {
